src/qibolab/serialize.py

```diff
@@ -119,8 +119,11 @@ def dump_qubits(qubits: QubitMap, pairs: QubitPairMap, couplers: CouplerMap = No
 
 def dump_instruments(instruments: InstrumentMap) -> dict:
     """Dump instrument settings to a dictionary following the runcard format."""
+    # Qblox modules settings are dictionaries and not dataclasses
```

Comment:
About this I'm not sure I understood your comment. So for now I put this patch that works both for qblox and the other instruments, but you want that I open an issue in order to migrate in the future the settings of qblox from dictionaries to dataclasses?

Reply:
Indeed, that's exactly the request.

The idea is that we were migrating to dataclasses, and that is the interface we'd like to maintain. Fine to have a temporary patch, but (at least in the future) Qblox should implement the interface, instead of just giving up on the interface we established.

Reply:
Ok perfect thanks. I will open an issue about that.

Reply:
Thanks